app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AccountActivationsController < ApplicationController
+  def edit
```

Comment:
I think method name should be ``activate``

Reply:
So, what's the problem to have the same here? It would be much more clearly to understand.